manifests/s/Shemeshg/MidiRouterClient/2.0.13/Shemeshg.MidiRouterClient.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Shemeshg.MidiRouterClient
+PackageVersion: 2.0.13
+InstallerType: nullsoft
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.sourceforge.net/midi-router-client/midi-router-client-2.0.13-win64.exe
+  InstallerSha256: 4BE088A94604000FFF77FEEE4978C29263C8B22D2ECD3560CF6024DCE5DEDD65
+- Architecture: arm64
+  InstallerUrl: https://downloads.sourceforge.net/midi-router-client/midi-router-client-2.0.13-win64-ARM.exe
+  InstallerSha256: 851866EBC6B2ED4898965171E6AAAB9E72D9A6ECCE2CA4CC48AF789E48812892

```

Comment:
```suggestion
  InstallerSha256: 851866EBC6B2ED4898965171E6AAAB9E72D9A6ECCE2CA4CC48AF789E48812892
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.arm64
```

Reply:
Hi it is true, I don't think I should manually deploy these files on may `cpack -G NSIS`
it is really a standard Qt c++ app not external dll except the standard...  

what would you recommand?

Reply:
ph, I see, it is just one DLL, I will do it on my CMake, I'll take me two days to setup a test machine.

Thanks, you've really helped